zeppelin-zengine/src/test/java/org/apache/zeppelin/conf/ZeppelinConfigurationTest.java

```diff
@@ -46,66 +46,66 @@ public void getAllowedOrigins2Test() throws MalformedURLException {
 
     ZeppelinConfiguration conf = ZeppelinConfiguration.create("test-zeppelin-site2.xml");
     List<String> origins = conf.getAllowedOrigins();
-    Assert.assertEquals(2, origins.size());
-    Assert.assertEquals("http://onehost:8080", origins.get(0));
-    Assert.assertEquals("http://otherhost.com", origins.get(1));
+    Assertions.assertEquals(2, origins.size());
```

Comment:
Trivia. I would prefer just `assertEquals` without `Assert` or `Assertion`.
You have already imported assertEquals statically above.